txircd/user.py

```diff
@@ -844,6 +835,17 @@ def modeString(self, toUser):
 			return "{} {}".format("".join(modeStr), " ".join(params))
 		return "".join(modeStr)
 
+	def resolveDNS(self, ip):
```

Comment:
This doesn't really have much to do with the user. Should it be in utils?

If nothing else, ircd might even be a better place for it than user.